Move to openspout/openspout + php 8.1 support

henryavila opened the pull request:
- Move from box/spout to openspout/openspout v3 | https://github.com/openspout/openspout
- Fix phpunit tests
- Add support to php 8.1
- Integrated with GitHub Actions
- Add auto dependabot merge

**Ps: All Unit Testes are passing**

rap2hpoutre commented:
Thank you for your contribution. I did not checked yet but do you know why all 4 automated process of tests fails?

henryavila commented:
> Thank you for your contribution. I did not checked yet but do you know why all 4 automated process of tests fails?

- **STYLECI** it's because some specific style from STYLECI

- **scrutinizer** fails because it config don't works on php 8.1 that was added in this version. PHP 8.1 requires openssl >= 1.0.2, but 1.0.1 is provided.

- How your plugin support php 7.1+ I added all versions from php 7.1 to 8.1. But the test on this versions is not working on **Travis CI**. Just the last php version (8.1) works. It's something related to `composer.lock`, since it's build on php 8.1, looks like Travis is not accepting another version (what make no sense, because it shoul get the proper version from `composer.json`)

- *CODACY* fails because of the text style.


Since I don't work with these tools, I can't fix their setup
But I've added in this PR the phpunit tests on GitHub Actions. On every PR and Commit, the phpunit will be executed on all php version from 7.1 to 8.1.
As I actualy works with GitHub Actions, any problem on this tests, I will be able do solve, but the others tools, unfortunately, I can't

rap2hpoutre commented:
@henryavila Thank you for your contribution. It seems it fails with github actions: https://github.com/rap2hpoutre/fast-excel/runs/7320967505
